kernel/kernel-notification-service/pom.xml

```diff
@@ -485,10 +485,12 @@
 				<dependency>
 					<groupId>io.mosip.kernel</groupId>
 					<artifactId>kernel-auth-adapter</artifactId>
-                                        <version>v1.2.0.1-B2</version>  <!-- Updating to last released version to remove circular dependency in release -->
+					<!--					Below version is only used for openapi-doc-generate-profile to generate api-docs-->
```

Comment:
Can you also say it will required only when running the below command to generate the api-docs and it is not required to have the latest version of auth adapter

Reply:
Done.